src/documenti/esterni/AdR/contenuti/casi_uso.tex

```diff
@@ -142,27 +142,100 @@ \subsection{UC5.4 - Selezione Sankey Diagram}
 \end{itemize}
 
 
-\section{UC6 - Filtri sui dati}
+\section{UC6 - Personalizzazione visualizzazione scelta}
 \begin{figure}[H]
   \centering
   \includegraphics[width=\textwidth]{uc6.png}
-  \caption{UC6 - Filtri sui dati}
+  \caption{UC6 - Personalizzazione visualizzazione}
 \end{figure}
 
 \begin{itemize}
-  \item \textbf{Descrizione}: l'utente ha la possibilità di modificare vari aspetti visivi del grafico;
-  \item \textbf{Attore primario}: utente;
+  \item \textbf{Descrizione}: l'utente ha la possibilità di modificare vari aspetti del grafico;
+  \item \textbf{Attore primario}: l'utente;
   \item \textbf{Precondizioni}: l'utente ha selezionato la tipologia di grafico [UC5] e l'applicativo lo ha generato;
   \item \textbf{Postcondizioni}: le modifiche apportate al grafico vengono visualizzate;
-  \item \textbf{Scenario principale}:
+  \item \textbf{Scenario principale}: 
+   \begin{enumerate}
+    \item L'utente può impostare varie opzioni del grafico scelto;
+    \item Le opzioni scelte vengono visualizzate;
+  \end{enumerate}
+  \item \textbf{Generalizzazioni}:
     \begin{enumerate}
-      \item L'utente può impostare i filtri da una lista automaticamente generata e il grafico viene visualizzato con le nuove caratteristiche;
+      \item Personalizzazione \textit{Scatter Plot} [UC6.1];
+      \item Personalizzazione \textit{Parallel Coordinates} [UC6.2];
+      \item Personalizzazione \textit{Force-Directed Graph} [UC6.3];
+      \item Personalizzazione \textit{Sankey Diagram} [UC6.4].
     \end{enumerate}
-  \item \textbf{Estensioni}:
+\end{itemize}
+
+\subsection{UC6.1 - Personalizzazione Scatter Plot}
+\subsection{UC6.2 - Personalizzazione Parallel Coordinates}
+
+\subsection{UC6.3 - Personalizzazione Force-Directed Graph}
+\begin{figure}[H]
+  \centering
+  \includegraphics[width=\textwidth]{uc6.3.png}
```

Comment:
in questo grafico credo che il frame debba avere come titolo uc6.3, quindi togliere il caso d'uso 6.3 che è il contorno e tu tratti al suo interno solo i suoi sottocasi. Per capirci come ha fatto codebuster [UC8.1 della RA]